TINY-6611: Check for empty url before calculating image dimensions (#6228)
tinymce/tinymce: modules/tinymce/changelog.txt

Version 5.7.0 (TBD)

>     Fixed an issue where image dimensions were trying to be calculated for an empty image URL #TINY-6611

Reviewer: ```suggestion
    Fixed an issue where the image dialog tried to calculate image dimensions for an empty image URL #TINY-6611
```
"trying to be" seemed not great to use in the changelog to me (considering we don't allow "being"), so switched things around a bit.

Author: cool, thanks

Version 5.7.0 (TBD)

>     Checking for empty url before calculating image dimensions #TINY-6611

Reviewer 2: `Checking` isn't a valid changelog entry. It must be one of the following:
- Added
- Changed
- Deprecated
- Fixed
- Removed

Author: Done

>     Fixed an issue where image dimensions were trying to be calculated for empty image url #TINY-6611

Reviewer 2: ```suggestion
    Fixed an issue where image dimensions were trying to be calculated for an empty image URL #TINY-6611
```
